luaui/Widgets_BAR/cmd_select_commander.lua

```diff
@@ -0,0 +1,28 @@
+function widget:GetInfo()
+  return {
+    name = "SelectCommander",
+    desc = "Press ctrl+c to select the commander",
+    author = "Teifion",
+    date = "26/10/2020",
+    license = "GNU GPL, v2 or later",
+    layer = 0,
+    enabled = true
+  }
+end
+
+function widget:Update()
+  local t = Spring.GetGameSeconds()
+  if Spring.GetSpectatingState() then
+    widgetHandler:RemoveWidget(self)
+    return
+  end
+  if (t > 0) then
+    local unitArray = Spring.GetTeamUnits(Spring.GetMyTeamID())
+    for key, unit in pairs(unitArray) do
+      if (unit.customParams.iscommander == true) do
```

Comment:
Isn't `unit` actually a unit ID (and not a unit def)?

```suggestion
      local unitDefID = Spring.GetUnitDefID(unit)
      local unitDef = UnitDefs[unitDefID]
      if (unitDef.customParams.iscommander == true) do
```